### PR TITLE
WAZO-3944-outgoing-caller-id-format

### PR DESCRIPTION
**Pull request by pc-m**
Depends-On: https://github.com/wazo-platform/xivo-dao/pull/285
Depends-On: https://github.com/wazo-platform/xivo-manage-db/pull/277

### CODE REVIEW

**Comment by pc-m** [comment]
recheck

File: wazo_confd/plugins/trunk/api.yml [hunk 1]
```diff
@@ -113,6 +113,18 @@ definitions:
...
+          default: +E164
+          enum:
+            - +E164
+            - E164
+            - national
+          description: |
+            This is the format that is going to be used when sending a caller ID to the trunk
+            operator. This formatting is only done for caller ID specified using the
+            `X-Wazo-Selected-Caller-ID` SIP header on an `INVITE`. If the user supplied caller ID
+            cannot be formated the requested caller ID will be ignored
```
Comment (DrPyser) [inline wazo_confd/plugins/trunk/api.yml:127]:
```suggestion
            The format to use for the caller ID of outgoing calls through this trunk, when the caller id is provided by an application through the `X-Wazo-Selected-Caller-ID` SIP header on an `INVITE`. 
            If the caller ID supplied by an application cannot be formatted, the requested caller ID will be ignored.
```

**Comment by pc-m** [comment]
recheck

**Review by DrPyser — APPROVED** [review]
LGTM, manually tested to be functional

**Comment by DrPyser** [comment]
recheck

**Pull request merged**